OpenRA.Game/Server/VoteKickTracker.cs

```diff
@@ -0,0 +1,194 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using OpenRA.Network;
+
+namespace OpenRA.Server
+{
+	public sealed class VoteKickTracker
```

Comment:
Why is this not a ServerTrait in `OpenRA.Mods.Common.Server`?
I feel it would make much more sense.

Also to implement `OpenRA.Server.ITick` instead of having a random `Tick()` method and implementing `ITick` on `LobbyCommands`, where it doesn't make much sense.

Reply:
It's a server helper class, one of many of its type. Though I suppose it could become a server trait. It will just be an iffy implementation as the server needs to hold a 1 public reference to it for UnitOrders to access.

It may be better to wait for the server refactor before doing this, unless @teinarss thinks otherwise

Reply:
But it would then need to be a static class as everything else is static and the only object is the server

Reply:
> LobbyCommands is way too overcrowded as it is